pkg/kubelet/kubelet_pods.go

```diff
@@ -207,6 +217,26 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 	return mounts, nil
 }
 
+// translateMountPropagation transforms v1.MountPropagationMode to
+// runtimeapi.MountPropagation.
+func translateMountPropagation(mountMode *v1.MountPropagationMode) (runtimeapi.MountPropagation, error) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.MountPropagation) {
+		// mount propagation is disabled, use private as in the old versions
+		return runtimeapi.MountPropagation_PROPAGATION_PRIVATE, nil
+	}
+	switch {
```

Comment:
If the feature is enabled, there is no way to specify `PRIVATE` and the default changes to HOST_TO_CONTAINER. 

This *might* be fine for Alpha, but can we actually change the default mode in Beta/GA? Sorry I didn't follow the PR proposal closely, so most likely this has been answered before.

Reply:
The proposal expects that HOST_TO_CONTAINER is the new default and PRIVATE is not needed at all. We need to test this theory in alpha.

Reply:
I'm slightly concerned that we won't gain enough confidence through feedback of this alpha feature. Changing the default behavior is always tricky.
I don't want to block the PR since the proposal has been through rounds of reviews. We can come back to this before graduating the feature to beta.